tripal/tripal
# Adds support for advanced sidebar and vertical tabs

# New Feature

### Issue #2007 


### Tripal Version: 4

## Description


This PR adds support for the following field group types: "Details Sidebar", "Tabs", "Tab".

## Testing?



1. Copy these files, change the file suffix from `.txt` to `.yml` and into your `tripal_layout/config/install` directory. This will replace existing files -do not commit them: [tripal_layout.tripal_layout_default_form.general_form.txt](https://github.com/user-attachments/files/17624673/tripal_layout.tripal_layout_default_form.general_form.txt) and [tripal_layout.tripal_layout_default_view.general_view.txt](https://github.com/user-attachments/files/17625081/tripal_layout.tripal_layout_default_view.general_view.txt)
2. Rebuild the cache
3. Go to Top Toolbar > Tripal > Page Structure > Analysis > Manage Form Display and click "Apply Tripal Layout".
![image](https://github.com/user-attachments/assets/8c777612-0960-4b7a-8447-d23a8a6520f2)
4. Now go to Manage Display for Analysis and click "Apply Tripal Layout".
![image](https://github.com/user-attachments/assets/fe26469f-b9a8-443a-9872-b91bc1056833)
5. Now go to Top Toolbar > Tripal > Content and click "Add Tripal Content" then "Analysis". Ensure you see the author in the sidebar and the vertical tab layout.
![image](https://github.com/user-attachments/assets/0461900d-674f-4a3c-9429-fbe28ed78258)
6. Fill out some content and confirm the vertical tabs display of the view page as well.
![image](https://github.com/user-attachments/assets/d22047b7-2883-462f-bf05-144d46b91e95)
